forge.config.ts

```diff
@@ -21,7 +21,7 @@ import type { ForgeConfig } from '@electron-forge/shared-types';
 const config: ForgeConfig = {
 	packagerConfig: {
 		asar: true,
-		extraResource: [ './wp-files', './assets' ],
+		extraResource: [ './wp-files', './assets', './bin' ],
```

Comment:
This copies both `bin/wp` (macOS) and `bin/wp.bat` (Windows). Ideally, we copy only the relevant file for the given platform.